File: User-Management-Api/user_management_api/user_management_repository/CustomViewModels/AddUserDetailsVm.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace user_management_repository.CustomViewModels
{
    public class AddUserDetailsVm
    {
        public string EmailId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public bool IsAdmin { get; set; }
        public string CreatedBy { get; set; }
        public int DeptId { get; set; }
    }
}

File: User-Management-Ui/src/app/shared/models/addUserDetailsVm.ts
export class AddUserDetailsVm{
    emailId: string;
    firstName: string;
    lastName: string;
    isAdmin: boolean;
    createdBy: string;
    puId: number;
    deptId: number;
    pdId: number;
    userRoles: UsersRoleVm[];
}

export class UsersRoleVm{
    roleId: number;
    roleName: string;
    description: string;
    isAssigned: boolean;
    isChanged: boolean;
}